src/Analyser/MutatingScope.php

```diff
@@ -674,6 +676,10 @@ private function getNodeKey(Expr $node): string
 			$key .= '/*' . $node->getAttribute('startFilePos') . '*/';
 		}
 
+		if ($node->hasAttribute(self::KEEP_VOID_ATTRIBUTE_NAME)) {
+			$key .= '/*' . self::KEEP_VOID_ATTRIBUTE_NAME . '*/';
```

Comment:
This should also check the value of the attribute and do it only if it's true.

Reply:
Good catch, thx